packages/lib/src/vis/utils.ts

```diff
@@ -309,3 +310,15 @@ export function projectCameraToHtml(
 
   return cameraVector.clone().applyMatrix4(cameraToHtmlMatrix);
 }
+
+export function isEventValid(
+  event: MouseEvent,
+  modifierKey: ModifierKey | undefined
+) {
+  if (!modifierKey) {
+    // If no modifier key is given, the event is deemed valid only if no modifier is are pressed
+    return !event.altKey && !event.ctrlKey && !event.shiftKey;
+  }
```

Comment:
If `modifierKey` is present, the event is valid if the key is pressed.

If `modifierKey` is not present, we expect the event to be valid for "simple" clicks (i.e. clicks with no modifier key pressed). I also considered using `!Object.values(ModifierKey).some((k) => event.getModifierState(k))` instead of the chained conditions.

Reply:
It's nice and clear the way it is (and allows using a type union).